chef/data_bags/secrets/production.json

```diff
@@ -1,68 +1,74 @@
 {
   "id": "production",
   "_password_comment": {
-    "encrypted_data": "qHlXr8rirW3ySV9z+pP7rYeGbnxCAcgLtv5vYZ6U/EeCMUsSQDeqM9lq0pHW\nCC+ZGlxJr9XY1EGZtKFUOLl4D7eUoC7c3rJu2hWfx8Pr4gEQB3AVsxa8XUaM\nIs+/C5VPB8LjIoTOBusYswrEMmVSRA==\n",
-    "iv": "ogGQgMdkrA7EzQXx0b6SEw==\n",
+    "encrypted_data": "/ce5YkvxK69ni8NFJWErjskGJfwmVwUyKu/yxEvrlq33CHZa3VeuNDgIUzFY\nd35YL2asqpFFOCBkSbVbEIcycD7zsuSTQkSFSiEmPEtTbLUbrNVKGPYD4CLF\nMvz45X+zwHjr7BAoSJ1BWZgx0Mof8A==\n",
```

Comment:
They actually didn't change, it's just a side effect of the encryption... when you change one, they all change.